projects/kit/components/input-phone/input-phone.component.ts

```diff
@@ -216,7 +216,9 @@ export class TuiInputPhoneComponent
     onValueChange(value: string): void {
         value = value === `` ? this.countryCode : value;
 
-        const parsed = isText(value) ? value : value.replace(TUI_MASK_SYMBOLS_REGEXP, ``);
+        const parsed = isText(value)
```

Comment:
Could you add unit test or cypress test?

Reply:
done 👍